src/App.js

```diff
@@ -1,15 +1,94 @@
 import React, { Component } from 'react'
 import ListArticles from './Components/ListArticles'
+import LoginForm from './Components/LoginForm'
+import { signInUser } from './state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
 
 class App extends Component {
+  state = {
```

Comment:
Is it possible to extract the entire Login functionality into a separate component as to avoid cluttering our App component? With redux, the state should be OK outside of the App component, right?
i.e.
class App...
return (
<SignUp />
)

class SignUp...
return (
 <LoginForm />
)

Reply:
We feel it's okay to have it in the App.js since we've used it before and we'd still need to send in data into that new component. It's not priority to refactor the code to that extent at the moment. But it is something we can revisit later if time allows.

Reply:
Since I am a minority on this case, I will accept. However, I am concerned about this becoming an issue later down the line f.eks. let's say the sign in functionality would require as much code as the login functionality, and you put all of it inside of the app component - would you still think it was easy to follow and track possible bugs? As opposed to knowing all of the code for the login-functionality is all extracted and restricted to one single component.

Reply:
We're thinking we'll cross that bridge when we get there :) But we agree it might be an issue later down the line